### PR TITLE
Integrate build-tools 2.1.1

### PR DESCRIPTION
**Pull request by romain-grecourt**

### CODE REVIEW

File: archetypes/catalog/cli-metadata.properties.mustache [hunk 1]
```diff
@@ -13,8 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-build-tools.version={{buildtoolsVersion}}
+build-tools.version=2.0.2
```
Comment (tomas-langer) [inline archetypes/catalog/cli-metadata.properties.mustache:16]: Seems to be the wrong version here, shouldn't this be the placeholder?

File: archetypes/catalog/cli-metadata.properties.mustache [hunk 2]
```diff
@@ -13,8 +13,9 @@
...
+cli.latest.plugin.version={{cliPluginVersion}}
 cli.version={{cliVersion}}
 {{#cliUpdateMessages}}
-cli.{{version}}.message={{message}}
+    cli.{{version}}.message={{message}}
```
Comment (tomas-langer) [inline archetypes/catalog/cli-metadata.properties.mustache:20]: Is the whitespace intentional?

File: pom.xml [hunk 3]
```diff
@@ -120,6 +120,8 @@
         <version.plugin.wiremock>2.14.0</version.plugin.wiremock>
         <version.plugin.echo>0.4.0</version.plugin.echo>
 
+        <version.helidon-cli>2.1.0</version.helidon-cli>
```
Comment (tomas-langer) [inline pom.xml:123]: Is the `cli` released separately? If not, this should be the same version as `build-tools`

**Review by tomas-langer — CHANGES_REQUESTED** [review]

**Comment by tomas-langer** [comment]
This PR is needed for `2.0.2` release, as otherwise the Helidon maven plugin fails to generate native image (at least on my machine for `mp-1` integration test).

File: archetypes/catalog/cli-metadata.properties.mustache (hunk 1 above)
Comment (barchetta) [reply archetypes/catalog/cli-metadata.properties.mustache:16]: Romain said this was intentional as it was "the version of the helidon-maven-plugin that contains the goal “dev” before we split it into helidon-cli-maven-plugin". And that this is used only for testing. We might want to follow-up on this, but I don't think it should block the PR.

File: pom.xml (hunk 3 above)
Comment (barchetta) [reply pom.xml:123]: No change was made to the CLI plugin, so he left this version the same.

File: archetypes/catalog/cli-metadata.properties.mustache (hunk 1 above)
Comment (romain-grecourt) [reply archetypes/catalog/cli-metadata.properties.mustache:16]: This property is correct. It corresponds to the older version of `helidon-maven-plugin` that contained the goal `dev`, it is used for backward compatibility. Also note that this file is only used for testing.

File: archetypes/catalog/cli-metadata.properties.mustache (hunk 2 above)
Comment (romain-grecourt) [reply archetypes/catalog/cli-metadata.properties.mustache:20]: Yes, this is a mustache loop ; thus the indentation.

File: pom.xml (hunk 3 above)
Comment (romain-grecourt) [reply pom.xml:123]: The CLI wasn't released as part of 2.1.1 because no changes were made to the CLI ; thus the difference in versions.

**Pull request merged**